litespi/phy/generic.py

```diff
@@ -78,12 +73,30 @@ def shift_out(self, width, bits, next_state, trigger=None, op=None, ddr=False):
         return res
 
     def __init__(self, pads, flash=None, device="xc7"):
+        assert flash is not None
```

Comment:
I think we can remove the `assert` statement and remove `flash` default value to get the same result